### DISCUSSION TITLE
Searching for an action that blocks merging based on labels

### DISCUSSION THREAD

**Discussion by Andre601** [community/community#27109] [Actions]
<p>Title sums it up pretty well.</p>
<p>I’m currently looking for a GitHub Action that would either return a success or failure depending on what labels have been added/removed from the Pull request.<br>
For context: I’m part of a Repository where we have a specific PR setup where draft PRs are seen as “not ready”, normal PRs are seen as “ready for review, but not mergable” and PRs with the “Status: Ready” label are also mergable.</p>
<p>So now would I like to use/setup a Action that would return a successful check/run when the PR has the “Status: Ready” label, therefore allowing you to merge the PR, and otherwise have it set to/return false to prevent the merging from happening.</p>
<p>Is there something like that?</p>

**Comment by mheap** [comment, marked as answer]
<p>You might find <a href="https://github.com/marketplace/actions/require-labels" class="inline-onebox" rel="noopener nofollow ugc">Require Labels · Actions · GitHub Marketplace · GitHub</a> useful</p>
